packages/venia-concept/src/components/Checkout/editableForm.js

```diff
@@ -23,34 +23,37 @@ const EditableForm = props => {
     } = props;
 
     const handleCancel = useCallback(() => {
-        editOrder(null);
-    }, [editOrder]);
+        setEditing(null);
+    }, [setEditing]);
 
     const handleSubmitAddressForm = useCallback(
-        formValues => {
-            submitShippingAddress({
+        async formValues => {
```

Comment:
We had to make some of the callbacks `async` so that they could appropriately handle an error response.